kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/logicaltype/DebeziumLogicalConverters.java

```diff
@@ -202,4 +204,23 @@ public String convert(Object kafkaConnectObject) {
       return bqZonedTimestampFormat.format(parsedTime);
     }
   }
+
+  /**
+   * Class for converting Debezium JSON data types to BigQuery JSON data types.
+   */
+  public static class JsonConverter extends LogicalTypeConverter {
+    /**
+     * Create a new ZoneTimestampConverter.
```

Comment:
to adjust `Create a new ZoneTimestampConverter.`

Reply:
fixed